sqrt_decomposition/sqrt_decomposition_simple.go

```diff
@@ -0,0 +1,105 @@
+package sqrt_decomposition
+
+import "math"
+
+/*
+You can use SqrtDecompositionSimple only if:
+
+Given a function $Query:E_1,...,E_n\rightarrow Q$
+
+if $\exist unionQ:Q,Q\rightarrow Q$
+
+s.t.
+
+- $\forall n\in \N > 1, 1\le i<n, E_1,..., E_n\in E \\ query(E_1,..., E_n)=unionQ(query(E_1,..., E_i), query(E_{i+1},...,E_n))$
+
+- (Only if you want use $update$ function)
+$\forall n\in \N > 0, E_1,..., E_n\in E \\ query(E_1,...,E_{new},..., E_n)=updateQ(query(E_1,...,E_{old},...,E_n), indexof(E_{old}), E_{new})$
+
+
+Usage example:
+	slice := []int32{1, 2, 3, 4, 5, 6}
+	s := NewSqrtDecompositionSimple(slice,
+		func(e int32) int32 { return e },
+		func(q1, q2 int32) int32 { return q1 + q2 },
+		func(q, a, b int32) int32 { return q - a + b },
+	)
+	s.Update(3, 10)
+	println(s.Query(1, 6)) // 26
+*/
```

Comment:
The commenting style is not following the Go standards defined in the `gofmt` program, please fix these issues.